app/controllers/autotune/blueprints_controller.rb

```diff
@@ -75,6 +75,18 @@ def destroy
       end
     end
 
+    def update_project_data
```

Comment:
should prolly name this something more explicit, like `build_data` or something or `preview_build_data`